### PR TITLE
Add templateUrl to the DeliverablePayload

### PR DESCRIPTION
**Pull request by nickgraz**

### CODE REVIEW

**Comment by nickgraz** [comment]
Current dependencies on/for this PR:
* **#1704** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/1704?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/1704?utm_source=stack-comment).

File: src/main/kotlin/com/terraformation/backend/accelerator/api/DeliverablesController.kt [hunk 1]
```diff
@@ -134,6 +134,7 @@ class DeliverablesController {
             projectId = ProjectId(3),
             projectName = "Omega Project",
             status = SubmissionStatus.Rejected,
+            templateUrl = "https://docs.google.com/document/d/1NkNoho843CE-6NM8rmmHfdAT7HJS6vWhfzezcvkj3yM/edit?usp=sharing",
```
Comment (sgrimm) [inline src/main/kotlin/com/terraformation/backend/accelerator/api/DeliverablesController.kt]: Remember that this is an open-source project (including the PR history) -- is that a document we want the world to know about?

**Review by sgrimm — APPROVED** [review]
Change is fine, but the example might not be.

File: src/main/kotlin/com/terraformation/backend/accelerator/api/DeliverablesController.kt (hunk 1 above)
Comment (sgrimm) [reply src/main/kotlin/com/terraformation/backend/accelerator/api/DeliverablesController.kt]: If it's just a template, the answer might be "yes" -- just making sure.

Comment (nickgraz) [reply src/main/kotlin/com/terraformation/backend/accelerator/api/DeliverablesController.kt]: It is a (near) empty Google Doc so I think it is alright.

Comment (nickgraz) [reply src/main/kotlin/com/terraformation/backend/accelerator/api/DeliverablesController.kt]: I replaced it with a public URL. @karthikbtf you may want to delete that test doc, you could potentially get random share requests, sorry about that.

Comment (karthikbtf) [reply src/main/kotlin/com/terraformation/backend/accelerator/api/DeliverablesController.kt]: Thanks for calling this out. It is an empty doc I created just for this purpose, was aware of the open source situation. I will delete it.

**Pull request merged**